config/htcondor-ce.spec

```diff
@@ -418,16 +445,21 @@ fi
 %{_datadir}/condor-ce/config.d/02-ce-bosco-defaults.conf
 
 %files client
+%config(noreplace) %{_sysconfdir}/condor-ce/condor_mapfile.osg
+%config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-osg.conf
```

Comment:
The original mapfile was in the client as well, though I'm not entirely sure why. The actual CE pulls in the client by default so it's not like you'd be able to run a CE without the mapfile.